stryker4jvm-mutator-kotlin/build.gradle.kts

```diff
@@ -1,31 +1,33 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
+group = "io.stryker-mutator"
+version = "1.0"
+
 plugins {
-    kotlin("jvm") //version "1.5.10"
-    kotlin("plugin.serialization") version "1.5.10"
+    kotlin("jvm") version "1.4.20"
```

Comment:
Is a downgrade necessary here?

Reply:
yessir, that is the reason why we get runtime exceptions. Version 1.5 and above does not allow illegal reflections which stryker4k depends on